portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase

```diff
@@ -239,4 +239,21 @@ definition {
 		WorkflowMetrics.verifyFiltersFromDashboard(isFilterOptionPresent = "Completed,Last 30 Day,Untracked");
 	}
 
+	@description = "View that Custom Range is selected with shown input date"
+	@priority = "5"
+	test VerifyCustomRangeDateInput {
+		property portal.acceptance = "true";
+
+		WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
+
+		Click(locator1 = "WorkflowMetrics#METRICS_PERFORMANCE_TAB");
+
+		WorkflowMetrics.assertDefaultChartRangesAndUnits();
+
+		WorkflowMetrics.selectDateRange(
+			completionVelocityDateRange = "Custom Range",
+			endDateRange = "11/11/2018",
+			startDateRange = "09/22/2018");
```

Comment:
@joshitagaki , when I was thinking about this test, I was also thinking about having a completed instance (blog, for instance) and check if the filter would also retrieve the correct value. So, in order to not change your script too much, you can do the following:

1. Create a blog instance and complete it. 
2. Filter for the same date range (no need to change it) -  from 09/22/2018 to 11/11/2018
3. Assert that the result is "0 inst / week"